### DISCUSSION TITLE
Moveflow --- 基于 Move 合约的流支付工具

### DISCUSSION THREAD

**Discussion by bengbengle** [NonceGeek/Move-Camp#183] [Amazing Things!]
一句话：做个move版本流支付工具，定位 Move 版本 superfluid

需要： 产品设计，合约开发，前端开发

域名:  moveflow.io 

联系方式：601544193（微信）

最简功能:
1.   连接钱包
2.   质押代币
3.   发送流支付
4.   时时显示流支付

#组队

**Comment by leeduckgo** [comment]
写得更详细一些。

**Comment by bengbengle** [comment]
Done 
https://github.com/Move-Stream/movecontract
https://github.com/Move-Stream/simple-dapp-demo
@leeduckgo

**Reply by albinlau** [reply]
👍👍👍
